<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-12-07T20:00:37Z" name="Suite" finished-at="2019-12-07T20:01:31Z" duration-ms="53907">
    <groups>
    </groups>
    <test started-at="2019-12-07T20:00:37Z" name="Test" finished-at="2019-12-07T20:01:31Z" duration-ms="53907">
      <class name="techm.techm1.test1">
        <test-method is-config="true" signature="launchDriver()[pri:0, instance:techm.techm1.test1@71def8f8]" started-at="2019-12-08T01:30:38Z" name="launchDriver" finished-at="2019-12-08T01:30:48Z" duration-ms="10030" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchDriver -->
        <test-method signature="Test1()[pri:0, instance:techm.techm1.test1@71def8f8]" started-at="2019-12-08T01:30:48Z" name="Test1" finished-at="2019-12-08T01:30:49Z" duration-ms="1719" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
        <test-method is-config="true" signature="launchDriver()[pri:0, instance:techm.techm1.test1@71def8f8]" started-at="2019-12-08T01:30:49Z" name="launchDriver" finished-at="2019-12-08T01:30:59Z" duration-ms="9289" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchDriver -->
        <test-method signature="Test2()[pri:0, instance:techm.techm1.test1@71def8f8]" started-at="2019-12-08T01:30:59Z" name="Test2" finished-at="2019-12-08T01:31:13Z" duration-ms="14073" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
        <test-method is-config="true" signature="launchDriver()[pri:0, instance:techm.techm1.test1@71def8f8]" started-at="2019-12-08T01:31:13Z" name="launchDriver" finished-at="2019-12-08T01:31:22Z" duration-ms="9708" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchDriver -->
        <test-method signature="Test3()[pri:0, instance:techm.techm1.test1@71def8f8]" started-at="2019-12-08T01:31:22Z" name="Test3" finished-at="2019-12-08T01:31:31Z" duration-ms="9061" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [AutomationTalks - Learn Automation Testing] but found [Expected title]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [AutomationTalks - Learn Automation Testing] but found [Expected title]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:171)
	at org.testng.Assert.assertEquals(Assert.java:181)
	at techm.techm1.test1.Test3(test1.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test3 -->
      </class> <!-- techm.techm1.test1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
